### PR TITLE
Easi 2610/model plan to user table

### PR DESCRIPTION
**Pull request by StevenWadeOddball**
# EASI-2610

## Changes and Description

- Model Plan now uses the user table
- DB seed refactored to more easily allow for migration
- Plan Favorite, exposed the user account GQL

## How to test this change
1. run scripts/dev db:seed
   a. ensure data is is seeded as expected
2. Update model plans using the front end and make sure that it is working as expected.

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

File: pkg/graph/resolvers/model_plan.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ import (
 // TODO Revist this function, as we probably want to add all of these DB entries inthe scope of a single SQL transaction
```
Comment (OddTomBrooks) [inline pkg/graph/resolvers/model_plan.go:20]:
```suggestion
// TODO: Revisit this function, as we probably want to add all of these DB entries in the scope of a single SQL transaction
```

File: pkg/models/model_helpers.go [hunk 2]
```diff
@@ -1,6 +1,9 @@
 package models
 
-import "github.com/lib/pq"
+import (
+	"github.com/google/uuid"
+	"github.com/lib/pq"
+)
 
 // StringPointer returns a pointer to a string input
 func StringPointer(st string) *string {
```
Comment (OddTomBrooks) [inline pkg/models/model_helpers.go:9]: Why not simply use &st?

**Review by OddTomBrooks — CHANGES_REQUESTED** [review]

File: pkg/graph/resolvers/model_plan.go (hunk 1 above)
Comment (StevenWadeOddball) [reply pkg/graph/resolvers/model_plan.go:20]: Hi @OddTomBrooks , that was a note from a previous time, not from this ticket, I don't this is the ticket to address it

File: pkg/models/model_helpers.go (hunk 2 above)
Comment (StevenWadeOddball) [reply pkg/models/model_helpers.go:9]: Hi @OddTomBrooks , this is also not a change from this ticket, and if we changed it it should be in a different ticket.

File: pkg/graph/resolvers/model_plan.go (hunk 1 above)
Comment (OddTomBrooks) [reply pkg/graph/resolvers/model_plan.go:20]: Fair enough. Bad formatting on TODO: may skip IDEs from picking them up so it threw a large red flag for me.

File: pkg/models/model_helpers.go (hunk 2 above)
Comment (OddTomBrooks) [reply pkg/models/model_helpers.go:9]: Fair enough. 👍

**Review by OddTomBrooks — APPROVED** [review]

**Pull request merged**